cypress/e2e/main.spec.js

```diff
@@ -434,6 +434,16 @@ describe('Scroll Plugin', function () {
 		cy.shouldHaveElementInViewport('[data-cy=anchor-by-name]');
 	});
 
+	it('should prefer undecoded id attributes', function () {
+		cy.get('[data-cy=link-to-anchor-encoded]').click();
+		cy.shouldHaveElementInViewport('[data-cy=anchor-encoded]');
+	});
+
+	it('should accept unencoded anchor links', function () {
```

Comment:
I find the wording a bit confusing here. How about these two?

- "should prefer undecoded anchor links"
- "should accept decoded anchor links"